pkgs/servers/seafile-server/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, python3Packages, autoreconfHook
+, libuuid, sqlite, glib, libevent, libsearpc, openssl, fuse, libarchive, which
+, vala, cmake, oniguruma }:
+
+let
+  libevhtp = import ./libevhtp.nix {
+    inherit stdenv lib fetchFromGitHub cmake libevent;
+  };
+in stdenv.mkDerivation rec {
+  pname = "seafile-server";
+  version = "8.0.5";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seafile-server";
+    rev = "v${version}-server";
+    sha256 = "0yki9y5p33xbznjpawmsfm52qa8njlnzmnfqnfsk07iz53m58dyx";
```

```suggestion
    sha256 = "sha256-OBv0oPLC2g6ApgD9lzj027nGHt7vOjlF88oeSmEPAdk=";
```
I'm getting a hash mismatch here

---

Can confirm, @greizgh might just have forgotten to change the hash from a previous version, causing that to be silently picked from the store.

---

Upstream actually [updated](https://github.com/haiwen/seafile-server/releases/tag/v8.0.5-server) the tag on june 17th :disappointed:

---

Yeah I agree using a commit hash is better then